README.md

```diff
@@ -15,7 +15,7 @@ Sugarcoat was created to enable developers to produce rich UI documentation easi
 
 # Index #
 
-  - [Features](#features)
+  - [FAQ](#faq)
```

Comment:
The format for linking within the page is: [Link Name](#link)
The #link is the name of the header, dash-delimited (`-`).
In this case, #faq should remain #features